news/5908.feature

```diff
@@ -0,0 +1,2 @@
+Wheel builds will output the final filename and SHA256 hash of .whl
```

Comment:
You should use the imperative tone (see https://pip.pypa.io/en/stable/development/contributing/#news-entries ). Maybe something like this?

> Log the final filename and SHA256 hash of a ``.whl`` file when done building a wheel.

Reply:
I have addressed this with 09e3184